airflow_client/docs/DAGApi.md

```diff
@@ -549,25 +549,7 @@ with client.ApiClient(configuration) as api_client:
     api_instance = dag_api.DAGApi(api_client)
     dag_id = "dag_id_example" # str | The DAG ID.
     dag = DAG(
-        dag_id="dag_id_example",
-        root_dag_id="root_dag_id_example",
         is_paused=True,
-        is_subdag=True,
-        fileloc="fileloc_example",
-        file_token="file_token_example",
-        owners=[
-            "owners_example",
-        ],
-        description="description_example",
-        schedule_interval=ScheduleInterval(
-            type="CronExpression",
-            value="value_example",
-        ),
-        tags=[
-            Tag(
-                name="name_example",
```

Comment:
Same here

Reply:
This is weird, as the new example does not contain the `dag_id` so

Reply:
will try executing some of these before merging

Reply:
Indeed that's a bit strange. I wonder where the examples are generated from :(

Reply:
Ok, did a quick check and the change seems to be correct. In patch dag `is_paused` is the only editable field.

Reply:
Looks like the new version respects read-only fields? If it is then there's no need for my current PR that's changing things inside the model.